src/sinks/new_relic_logs.rs

```diff
@@ -13,12 +13,20 @@ use indexmap::IndexMap;
 use serde::{Deserialize, Serialize};
 use snafu::Snafu;
 
+// New Relic Logs API accepts payloads up to 1MB (10^6 bytes)
+const MAX_PAYLOAD_SIZE: usize = 1000000 as usize;
```

Comment:
```suggestion
const MAX_PAYLOAD_SIZE: usize = 1_000_000 as usize;
```

(Non blocking, nit)

Reply:
Also nit: I might throw a link in to https://docs.newrelic.com/docs/telemetry-data-platform/ingest-manage-data/ingest-apis/use-event-api-report-custom-events#limits.

Reply:
Personally, I would prefer not to include this link. The reason is that this documentation is written for the Event API, not for the Log API. Under the hood the rules are the same for both APIs, but this documentation talks explicitly of the Event API, which may be confusing to readers. Our docs team is working to provide the same information also in the Log API context.

Reply:
Ah, gotcha :+1: